javascript/webdriver/command.js

```diff
@@ -219,7 +219,9 @@ webdriver.CommandName = {
   GET_SESSION_LOGS: 'getSessionLogs',
 
   // Non-standard commands used by the standalone Selenium server.
-  UPLOAD_FILE: 'uploadFile'
+  UPLOAD_FILE: 'uploadFile', 
+
+  GET_CANVAS_URL: 'getCanvasUrl'
```

Comment:
This is Safari specific and should be defined somewhere in the `safaridriver` namespace